### PR TITLE
Cross-build for Native

### PR DESCRIPTION
**Pull request by armanbilge**
Not yet prime time, opening for CI :)

Update: published `3.2.1-33-97a8eb2` to Maven Central.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -44,10 +47,10 @@ lazy val docs = project
...
 
 val catsV = "2.8.0"
-val catsEffectV = "3.3.14"
-val disciplineMunitV = "1.0.9"
-val scalacheckEffectV = "1.0.4"
-val munitCatsEffectV = "1.0.7"
+val catsEffectV = "3.3.14-1-5d11fe9"
+val disciplineMunitV = "2.0.0-M3"
+val scalacheckEffectV = "2.0-9366e44"
+val munitCatsEffectV = "2.0-5e03bfc"
```
Comment (armanbilge) [inline build.sbt:53]:
```suggestion
val catsEffectV = "3.3.14"
val disciplineMunitV = "2.0.0-M3"
val scalacheckEffectV = "2.0.0-M2"
val munitCatsEffectV = "2.0.0-M3"
```

**Review by ChristopherDavenport — APPROVED** [review]

**Review by danicheg — APPROVED** [review]
🤯

**Comment by armanbilge** [comment]
Thanks @danicheg, would you mind cutting a release with this for me please :)

**Comment by danicheg** [comment]
@armanbilge here we go https://github.com/typelevel/vault/releases/tag/v3.3.0.

**Pull request merged**